include/bus_handler.hpp

```diff
@@ -65,6 +65,10 @@ class BusHandler
                                      }
                                      return 0;
                                  });
+        iface->register_method("TriggerPanelFunction",
```

Comment:
These functions have nothing to do with panel. 
Please expose a generic name - "TriggerFunction".

Reply:
better would be executeFunction

Reply:
done